requirements.txt

```diff
@@ -11,3 +11,4 @@ opencv-python==4.3.0.36
 Pillow>=8.2.0
 git+https://github.com/MartinSmeyer/cocoapi.git@v1.0#subdirectory=PythonAPI
 vispy>=0.6.5
+open3d==0.15.2
```

Comment:
since open3d is a huge library, lets rather remove it from the requirements and add manual installation instructions in the README

Reply:
also there is a conflict with `matplotlib==2.2.4`

Reply:
I removed Open3D from the requirements. I added instructions in the README to install it separately if the annotation tool is going to be used.